google/cloud/testing_util/opentelemetry_matchers.cc

```diff
@@ -164,6 +164,18 @@ std::string ToString(opentelemetry::trace::SpanContext const& span_context) {
   return ss.str();
 }
 
+std::string ToString(opentelemetry::trace::SpanId const& span_id) {
```

Comment:
Maybe we should refactor these functions to `internal/opentelemetry.h`?

Reply:
Done. Moved the functions to opentelemetry.h